### PR TITLE
Added info on log in to MP for 1st time

### PR DESCRIPTION
**Pull request by alextopuzov**
Part of clarifying the users story in 1.2.
Added a concept topic in the MP doc, added to Summary.

### CODE REVIEW

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Logging In to the Management Portal #
+
+You can access the Management Portal in a web browser by entering the vSphere Integrated Containers instance IP address and the port that you specified for the portal during the deployment. By default the port number is *8282*.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: "vSphere Integrated Containers instance IP address" --> "vSphere Integrated Containers appliance IP address"

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+If you don't know the port number, you can access the portal by going to http://<i>vic_appliance_address</i> and following the **Go to the vSphere Integrated Containers Management Portal** link.
+
+## Default User Access to the Management Portal ##
+
+The role that has full permissions for vSphere Integrated Containers is the cloud administrator role. 
+By default, the cloud administrator role is assigned to the Administrators group for vCenter Server during the installation of vSphere Integrated Containers. Every user that is a member of that group in the Platform Services Controller can access the Management Portal as cloud administrator. After you log in as a cloud administrator, you can give other users access to vSphere Integrated Containers by assigning them roles to projects.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: "by assigning them roles to projects" --> "by assigning them roles in projects"

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Optionally, you can log in with a user that you created during the OVA deployment, if you used that option.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: We need to confirm that this does indeed end up being the case.

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md (hunk 1 above)
Comment (alextopuzov) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: fixed

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md (hunk 2 above)
Comment (alextopuzov) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: fixed

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Logging In to the Management Portal #
+
+You can access the Management Portal in a web browser by entering the vSphere Integrated Containers appliance IP address and the port that you specified for the portal during the deployment. By default the port number is *8282*.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: The users who log into the MP are not necessarily going to be the same as the vSphere admin who deployed the OVA, so "the port that you specified" is probably not  going to be "you".  I think that we can bypass the whole problem of the OVA using non-default ports by just sending people to the Getting Started page and telling them to click the link. This way, we don't need to mention port numbers at all.

**Comment by stuclem** [comment]
I confirmed in today's build that the option to deploy the appliance with OOTB users is indeed implemented, so this is now good to go.

**Pull request merged**